cerbero/packages/package.py

```diff
@@ -421,6 +421,7 @@ class MetaPackage(PackageBase):
     platform_packages = {}
     sdk_version = '1.0'
     resources_wix_installer = None
+    wix_config_custom = None
```

Comment:
other vars related to wix are in the form foo_wix_bar, instead of wix_foo_bar, try to use the same variable scheme for this